### ISSUE TITLE
Broken link on Wiki page

### ISSUE THREAD

**Issue by shadrech** [chenglou/react-motion#472]
Link for the 'component-router' on the Wiki section 'Gallery of third party React Motion demos' is broken (link: http://in-flux.github.io/component-router/example/)

**Comment by haxxxton** [comment]
Correct link: [http://in-flux.github.io/component-router/](http://in-flux.github.io/component-router/)

**Comment by nkbt** [comment]
yeah, thanks. I also did not update component-router example to use react-motion... So probably better to remove link for now.
